tests/queries/0_stateless/02882_formatQuery.reference

```diff
@@ -0,0 +1,9 @@
+SELECT 1
+SELECT 1
+SELECT 1
+SELECT 1
+1
+INSERT INTO tab FORMAT Values
+CREATE TABLE default.no_prop_table\n(\n    `some_column` UInt64\n)\nENGINE = MergeTree\nORDER BY tuple()\nSETTINGS index_granularity = 8192
+EXPLAIN SYNTAX\nSELECT\n    CAST(1, \'INT\'),\n    CEIL(1),\n    CEILING(1),\n    CHAR(49),\n    CHAR_LENGTH(\'1\'),\n    CHARACTER_LENGTH(\'1\'),\n    COALESCE(1),\n    CONCAT(\'1\', \'1\'),\n    CORR(1, 1),\n    COS(1),\n    COUNT(1),\n    COVAR_POP(1, 1),\n    COVAR_SAMP(1, 1),\n    DATABASE(),\n    SCHEMA(),\n    dateDiff(\'DAY\', toDate(\'2020-10-24\'), toDate(\'2019-10-24\')),\n    EXP(1),\n    FLATTEN([[1]]),\n    FLOOR(1),\n    FQDN(),\n    GREATEST(1),\n    IF(1, 1, 1),\n    IFNULL(1, 1),\n    LCASE(\'A\'),\n    LEAST(1),\n    LENGTH(\'1\'),\n    LN(1),\n    LOCATE(\'1\', \'1\'),\n    LOG(1),\n    LOG10(1),\n    LOG2(1),\n    LOWER(\'A\'),\n    MAX(1),\n    MID(\'123\', 1, 1),\n    MIN(1),\n    MOD(1, 1),\n    NOT 1,\n    NOW(),\n    NOW64(),\n    NULLIF(1, 1),\n    PI(),\n    position(\'123\', \'2\'),\n    POW(1, 1),\n    POWER(1, 1),\n    RAND(),\n    REPLACE(\'1\', \'1\', \'2\'),\n    REVERSE(\'123\'),\n    ROUND(1),\n    SIN(1),\n    SQRT(1),\n    STDDEV_POP(1),\n    STDDEV_SAMP(1),\n    SUBSTR(\'123\', 2),\n    substring(\'123\', 2),\n    SUM(1),\n    TAN(1),\n    TANH(1),\n    TRUNC(1),\n    TRUNCATE(1),\n    UCASE(\'A\'),\n    UPPER(\'A\'),\n    USER(),\n    VAR_POP(1),\n    VAR_SAMP(1),\n    WEEK(toDate(\'2020-10-24\')),\n    YEARWEEK(toDate(\'2020-10-24\'))\nFORMAT TSVRaw
```

Comment:
This doesn't look like formatted query

Reply:
#27229 is about "normalized formatting", e.g.

```sql
select a,    b FRom tab WHERE a > 3 and  b < 3
```

should format into a "normal form" a la:

```sql
SELECT
    a, b
FROM
    tab
WHERE
    a > 3
    AND b < 3
```

(basically the formatting which clickhouse-client prints after each query and before it prints the result)

Reply:
Related to that, the docs should include some more complex example than `SELECT 1` for which formatting doesn't make a difference.